doc/neps/nep-0046-sponsorship-guidelines.rst

```diff
@@ -108,16 +108,13 @@ About page) will be added to acknowledge all current and previous sponsors,
 partners, and any other entities and individuals who provided $5,000 or more of
 financial or in-kind support. This page will include relevant details of
 support (dates, amounts, names, and purpose); no logos will be used on this
-page. The rationale for the $5,000 minimum level is to keep the amount of work
-maintaining the page reasonable; the level is the equivalent of, e.g., one GSoC
-or a person-week's worth of engineering time in a Western country, which seems
-like a reasonable lower limit.
-
-Additionally, accredidation for smaller focused support in implementing
-specific ehancements or fixes may be attributed in the release notes for that
-contribution. In this case the accredidation will take the form of something
-like "Thanks to <name and url> for sponsoring this work" in the appropriate
-release note blurb.
+page. Such support, if provided for a specific enhancements or fix, may be
+acknowledged in the release notes, through a "Thanks to
+``sponsor-name-and-url`` for sponsoring this work" in the appropriate release
+note snippet.  The rationale for the $5,000 minimum level is to keep the amount
```

Comment:
This is fine, but I'd word it even simpler:

> Such support, if provided for a specific enhancement, shall be acknowledged in the release notes.

I don't think we need to say how we are going to word it. If you feel it's important to add that, I'd simply use `sponsored by <>` in the relevant PR line.

Reply:
Done, please check.